webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatus.stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PowerStatus from './index';
+import {
+  Story,
+  Code,
+  Text,
+  StoryWithCustomState,
+} from '../../../../../../stories/components';
+import {
+  pendingStore,
+  resolvedStore,
+  resolvedStoreWithOff,
+  errorStore,
+  data,
```

Comment:
This var doesn't exist in the fixtures file

Reply:
Thanks, great catch !
while we are at it I refactored the server props.